sysend.js

```diff
@@ -124,8 +124,16 @@
     }
     if (is_iframe()) {
       window.addEventListener('message', function(e) {
-          var payload = JSON.parse(e.data);
-          if (payload.name === uniq_prefix) {
+          if (typeof e.data !== 'string') return;
+          var isValidJSON = false;
```

Comment:
The flag looks weird can you change it to make the broadcast inside try without the flag?

Reply:
I have removed it